ReactWindows/ReactNative/Views/TextInput/ReactClearButtonModeHelper.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file is named ...Helper.cs but other similar files containing enums are named after the enum. It may be better to just follow the pattern and name the file ClearButtonModeType.cs e.g. like ScrollEventType.cs <xxxxx>Helper files usually contain standalone methods related to the some functionality.

Reply:
Got it. Updating name.